packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -62,32 +62,32 @@ export const TranslationSidebar = ({
     id: language,
   });
 
-  // Only update the content shown in the text editor when the word ID changes.
-  const [originalNoteContent, setOriginalNoteContent] = useState('');
   const [noteContent, setNoteContent] = useState('');
   const wordId = useRef('');
   useEffect(() => {
-    if (word.id !== wordId.current) {
+    if (translatorNotesQuery.isSuccess && word.id !== wordId.current) {
```

Comment:
I also had to add `isSuccess` to this effect check otherwise it would run before the notes had loaded over the network